domains/libraries/EnergyOrigin.Domain/EnergyOrigin.Domain.ValueObjects.Tests/TinTest.cs

```diff
@@ -1,43 +1,43 @@
-using API.ValueObjects;
+using Xunit;
 
-namespace API.UnitTests.ValueObjects;
+namespace EnergyOrigin.Domain.ValueObjects.Tests;
 
-public class TinTests
+public class TinTest
 {
     [Fact]
     public void Tin_WithValidData_CreatesSuccessfully()
     {
-        var tin = new Tin("00000000");
+        var tin = Tin.Create("00000000");
         Assert.Equal("00000000", tin.Value);
     }
 
     [Fact]
     public void Tin_WithInvalidLength_ThrowsException()
```

Comment:
Minor: Make Theory and add test for less than 8 digits

Reply:
Added test test for less than 8 digits, the Theory part, do you mean to collapse all the tests into one With Theory cases?

Reply:
Nah just this test case

Reply:
Done